api/graphql/tests/test_reservations/test_reservation_queries.py

```diff
@@ -836,6 +836,136 @@ def test_filter_by_text_search_individual_reservee_name(self):
         assert_that(content.get("errors")).is_none()
         self.assertMatchSnapshot(content)
 
+    def test_order_by_reservation_unit_name(self):
+        resunitA = ReservationUnitFactory(
+            name_fi="a Unit", name_en="d Unit", name_sv="g unit"
+        )
+        resunitB = ReservationUnitFactory(
+            name_fi="b Unit", name_en="e Unit", name_sv="h unit"
+        )
+        resunitC = ReservationUnitFactory(
+            name_fi="c Unit", name_en="f Unit", name_sv="i unit"
+        )
+
+        ReservationFactory(name="this should be 1st", reservation_unit=[resunitA])
+        ReservationFactory(name="this should be 2nd", reservation_unit=[resunitB])
+        ReservationFactory(name="this should be 3rd", reservation_unit=[resunitC])
+
+        self.client.force_login(self.general_admin)
+        test_data = ["fi", "en", "sv"]
+        for lang in test_data:
+            response = self.query(
+                """
+                query {
+                    reservations(orderBy:"reservation_unit_name_%s") {
```

Comment:
There should be some magic in the graphene django code and this filter name should work as camel case (as we camelcase the api). So we could/should have tests in camelCases since its the way that the frontend is using this.
`reservations(orderBy: "reservationUnitNameFi"` etc.

Reply:
Learned something new today! 🧠 
I changed the tests so that they now use cameCase 🧹